### ISSUE TITLE
Make MicroWin perform certain tasks based on Image Type (Home, Pro, LTSC, etc...) & Image Language

### ISSUE THREAD

**Issue by og-mrk** [ChrisTitusTech/winutil#2869]
**Is your feature request related to a problem? Please describe.**

Currently MicroWin, _as far as I understand_, perform actions regardless of the Image Type, making it treat LTSC versions like it's Pro or Home Editions, and so on.. or have International Language Version of Windows be treated like other languages, as certain languages have some packages removed due to that're unknown to us, or for example, Microsoft complying to EU's Policies, or other reasons as well.

**Describe the solution you'd like**

I'm not experienced with MicroWin's Code base.. but, if it's possible to get an Image type & language, we could do a State Machine (or a Finite one) that'll do certain tasks for specific editions and languages.

**Describe alternatives you've considered**

There's probably other ways that I haven't thought of.. so this's up to the people who know what they're talking about (i.e. not me 😆)

**Additional context**

In the end.. this's just a suggestion, no pressure to do it if nobody wants this added complexity into MicroWin.

**Comment by FatBastard0** [comment]
Yeah I was thinking that maybe an toggle our button for an expert mode
I do agree with Mr. Titus, with the concept that MicrWin, should be very easy to you.
Me being new at game, I am that guy that would get lost on how to do something.

**Comment by holy-mother-I-am-drunk** [comment]
It would be really nice to have an "expert" toggle that lets us choose what to disable and what to keep.
Once toggled it will show more options to check for example:
To delete Ms store but to keep the gamebar as it's used by people to record and screenshot.
And in ltsc, AI is already disabled so it doesn't matter if microwin removes it again it's just redundant

**Comment by FatBastard0** [comment]
Good idea!
I would like to thank you in advance for any assistance that you can
provide in this matter. Please feel free to contact me at the below listed
address, Telephone and/or email.




Richard K. L'Italien
Disable Veteran of the US Army
The Fat Bastard
5637 Monte Corita Circle
Citrus Heights California 95621
(916) 915-3883 <http://voice.google.com/calls?a=nc,%2B19169153883>
<http://voice.google.com/calls?a=nc,%2B19169153883> Voice
(916) 943 4967 <http://voice.google.com/calls?a=nc,%2B19169434967>
<http://voice.google.com/calls?a=nc,%2B19167453437>

******************************************************************************
This communication (including any attachments) may contain privileged or
confidential information intended for a specific individual and purpose,
and is protected by law.  If you are not the intended recipient, you should
delete this communication and/or shred the materials and any attachments and
are hereby notified that any disclosure, copying, or distribution of this
communication, or the taking of any action based on it, is strictly
prohibited.

Thank you.

**Comment by ghost** [comment]
to hard to implament and would greatly increase the size of microwin even tho this isn't really needed /close
